expression/bench_test.go

```diff
@@ -373,6 +373,29 @@ func (g *randHexStrGener) gen() interface{} {
 	return string(buf)
 }
 
+type randSpaceStrGener struct {
```

Comment:
It is not a common generator, could it be put into builtin_string_vec_test.go

Reply:
okay.  I move it to builtin_string_vec_test.go